_docs/20-logappender-example.md

```diff
@@ -0,0 +1,80 @@
+---
+title: "Logappender Example"
+permalink: /docs/logappender-example/
+excerpt: "How to use logappender in RocketMQ."
+modified: 2017-06-08T21:01:43-04:00
+---
+
+{% include toc %}
+
+When we use rocketmq as a part of big data processing or other cases,bussiness also need to put log data to rocketmq.Rocketmq logappender provides log4j appender,logback appender and log4j2 appender for bussiness to use,below is some example.
```

Comment:
"Rocketmq logappender provides log4j appender,logback appender and log4j2 appender for bussiness to use,below is some example."--> RocketMQ log appender provides log4j, logback and log4j2 appenders for business developers. The following are some examples.